### ISSUE TITLE
TablePrimitive should accept `scope` as a prop

### ISSUE THREAD

**Issue by rachelslurs** [roninoss/rn-primitives#64]
I haven't gone through all of the table-related components yet, but I noticed I was unable to add `scope` because you don't accept any table accessibility-specific props. Is there a reason for this? I'm new to react-native so apologies if this is obvious. 🤪
Maybe I'm supposed to do it `asChild` for my use case and add whatever props I need in there?

https://github.com/roninoss/rn-primitives/blob/main/packages/table/src/table.tsx#L41-L44

Thank you in advance!
Rachel
